lib/milvus/search.rb

```diff
@@ -7,7 +7,7 @@ class Search < Base
     def post(
       collection_name:,
       anns_field:, top_k:, params:, metric_type:, vectors:, dsl_type:, output_fields: nil,
-      round_decimal: nil
+      round_decimal: nil, partition_names: nil,vector: nil, filter: nil
```

Comment:
@washu Could you please send me a link to the documentation where you found that these params are supported?

Reply:
you are correct i mis read a GRPC call for http api call. reverted this out